backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
```

Comment:
By default, Flyway always wraps the execution of an entire migration within a single transaction.

Reply:
Cool!